pkgs/applications/system/howdy/sources.nix

```diff
@@ -0,0 +1,32 @@
+{ fetchurl
+, fetchFromGitHub
+, python3
+, ...
+}: {
+  data = {
+    "dlib_face_recognition_resnet_model_v1.dat" = fetchurl {
+      url = "https://github.com/davisking/dlib-models/raw/master/dlib_face_recognition_resnet_model_v1.dat.bz2";
```

Comment:
Should not these be pinned to a commit in case it is modified?

Reply:
Sure, I will pin them.